.eslintrc

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "airbnb",
   "rules": {
+    "no-var": 0,
     "object-curly-spacing": 0,
-    "no-var": 0
+    "space-before-function-paren": 0
```

Comment:
I'm pro-`space-before-function-paren` (for anonymous functions) but understand that it is not generally our style.